modules/admin/lib/Controller/ConfigController.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-namespace SimpleSAML\Module\admin;
+namespace SimpleSAML\Module\admin\Controller;
```

Comment:
Maybe we can actually rename these controllers as `SimpleSAML\Module\admin\Controller\Config` and so on?

Reply:
Done